table/table_test.cc

```diff
@@ -2317,93 +2317,114 @@ std::map<std::string, size_t> MockCache::marked_data_in_cache_;
 // table is closed. This test makes sure that the only items remains in the
 // cache after the table is closed are raw data blocks.
 TEST_P(BlockBasedTableTest, NoObjectInCacheAfterTableClose) {
+  std::vector<CompressionType> compression_types{kNoCompression};
+
+#ifdef ZLIB
+  compression_types.push_back(kZlibCompression);
+#endif  // ZLIB
+#if LZ4_VERSION_NUMBER >= 10400
```

Comment:
I guess this logic is coped from somewhere. Would be great to put an inline comment, referencing that, so that one could go figure the rationale behind the magic numbers.

Reply:
Yep, it comes from the compression dictionary tests. Will add a comment to clarify that.